spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -719,6 +719,31 @@ describe('GOVUK.analyticsGa4.linkTracker', function () {
       }
     })
 
+    it('cleans up link text with multiple lines and spaces', function () {
+      linkTracker = GOVUK.analyticsGa4.analyticsModules.Ga4LinkTracker
+      linkTracker.init({ internalDomains: ['www.gov.uk'] })
+
+      links.innerHTML = '<div class="messy-text">' +
+      '<a href="https://example.com">\nI \nam \non \nmultiple \n\n\n lines</a>' +
+      '<a href="https://www.gov.uk/government/uploads/example.pdf">I     have     lots   of    spaces</a>' +
+      '<a href="mailto:hello@example.com">I     have     lots   of    spaces \n and \n\n many \n\n\n lines</a>' +
```

Comment:
Not sure how this actually works, but is it worth checking real line breaks? e.g.

```
'<a href="mailto:hello@example.com">I     have     lots   of    spaces \n and \n\n many ' + 
'many ' +
'lines</a>' +
```

Reply:
@andysellick I don't think JS counts line breaks outside of quotes. I just tried your string in jsFiddle and it doesn't add any line breaks for `many many lines</a>"`